src/Payum/Paypal/ExpressCheckout/Nvp/PaypalExpressCheckoutGatewayFactory.php

```diff
@@ -33,7 +33,20 @@ class PaypalExpressCheckoutGatewayFactory extends GatewayFactory
      */
     protected function populateConfig(ArrayObject $config)
     {
-        $config->defaults(array(
+        $config->defaults($this->getConfigDefaults());
+        $this->setDefaultApi($config);
+
+        $config['payum.paths'] = array_replace([
+            'PayumPaypalExpressCheckout' => __DIR__.'/Resources/views',
+        ], $config['payum.paths'] ?: []);
+    }
+
+    /**
+     * @return array
+     */
+    protected function getConfigDefaults()
```

Comment:
instead of these method could you please add a populateApiConfig(ArrayObject $config) method?